rosa_tutorials/rosa-mobb-sts-rds-database-connect.adoc

```diff
@@ -0,0 +1,399 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-sts-rds-database-connect"]
+= Tutorial: Connecting to RDS database with STS from ROSA
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-sts-rds-database-connect
+
+toc::[]
+
+// Mobb content metadata
+// Brought into ROSA product docs 2023-09-19
+// ---
+// date: '2023-07-24'
+// title: Connect to RDS database with STS from ROSA
+// tags: ["AWS", "ROSA", "RDS", "STS"]
+// authors:
+//  - Florian Jacquin
+// ---
+
+This guide shows how to quickly connect an Amazon Web Services Relational Database Service (AWS RDS) from Red Hat OpenShift Service on AWS (ROSA).
+
+The AWS RDS can be consumed from ROSA and authenticated to DB with Security Token Service (STS).
+
+
+== Amazon Web Services Relational Database Service
+
+Amazon Web Services Relational Database Service (AWS RDS) is a distributed relational database service by Amazon Web Services. It is designed to simplify setup, operation, and scaling of a relational database for use in applications. AWS RDS supports database engines such as Amazon Aurora, MySQL, MariaDB, Oracle, Microsoft SQL Server, and PostgreSQL.
+
+In our example we will use PostgreSQL as engine.
+
+== Prerequisites
+
+* A ROSA cluster on 4.12 or later
+* The OpenShift CLI
+* The AWS CLI
+* The `jq` command
+
+== Environment Setup
+Configure the following environment variables.
+
+* Export cluster information:
++
+[source,terminal]
+----
+$ (`rosa list cluster`)
```

Comment:
This line is meaningless and will cause the command fail.
If we want to show customer how to find the <cluster name>, move it out from current command text to a single command